### PR TITLE
Add a brief description on Choreo

### PR DESCRIPTION
**Pull request by Nashaath**
## Purpose
Add a brief description on Choreo.

### CODE REVIEW

File: en/docs/get-started/what-is-choreo.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
 
+Choreo, a versatile and comprehensive platform for low-code, cloud-native engineering, provides the following advantages:
+
+- An AI-assisted, low-code application development environment that can help developers build enterprise-class integrations faster while ensuring best practices and secure coding guidelines. 
+- Developers can switch between a visual low-code and typical code environment and select the experience that best supports a given situation.
+- Even without expert-level coding skills, Choreo's low-code environment empowers developers to build applications with ease. It also provides capabilities to collaborate and improve these applications as needed.
+- Its low-code application development platform auto-generates editable and version-controlled code and maintains it in GitHub. This enables applying the engineering rigor required for enterprise applications on low-code.
```
Comment (RukshiW) [inline en/docs/get-started/what-is-choreo.md:10]: Suggestion: Can we change "enables applying" to "enables you to apply"?

File: en/docs/get-started/what-is-choreo.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+- API management capabilities to develop APIs and get them to production. Choreo’s API Publisher Portal allows API developers to design, publish, and manage the lifecycle of APIs whereas the API Developer Portal allows consumers to consume APIs.
+- Realistic DevOps with versions, environments, CI/CD pipelines, and automation tests.
+- Deep observability to trace your executions from the code statement level to the machine it runs.
```
Comment (RukshiW) [inline en/docs/get-started/what-is-choreo.md:13]: Can we change "the machine it runs" to "the machine in which they run"?

File: en/docs/get-started/what-is-choreo.md (hunk 1 above)
Comment (Nashaath) [reply en/docs/get-started/what-is-choreo.md:10]: "enables you to apply" doesn't sound correct in this context. How about we change it to "makes it possible to apply"?

Comment (RukshiW) [reply en/docs/get-started/what-is-choreo.md:10]: Ok

File: en/docs/get-started/what-is-choreo.md (hunk 2 above)
Comment (Nashaath) [reply en/docs/get-started/what-is-choreo.md]: Changed this to "the machine on which those run"

**Review by RukshiW — APPROVED** [review]

**Pull request merged**